api/fees/types.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2025 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package fees
+
+import (
+	"sync"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/vechain/thor/v2/bft"
+	"github.com/vechain/thor/v2/cache"
+	"github.com/vechain/thor/v2/chain"
+)
+
+type Fees struct {
```

Comment:
Better move `Fees` into `fees.go` and same with `FeesData`, `FeeCacheEntry`.

Reply:
done